ts.js

```diff
@@ -124,7 +125,28 @@
          }
       ],
       '@typescript-eslint/no-inferrable-types': ['error', {'ignoreParameters': true}],
-      '@typescript-eslint/no-misused-promises': ['error', {'checksConditionals': true, 'checksVoidReturn': false}]
+      '@typescript-eslint/no-misused-promises': ['error', {'checksConditionals': true, 'checksVoidReturn': false}],
+      '@typescript-eslint/class-literal-property-style': 'error',
+      '@typescript-eslint/no-confusing-non-null-assertion': 'error',
+      '@typescript-eslint/no-extraneous-class': 'error',
+      '@typescript-eslint/no-meaningless-void-operator': 'error',
+      '@typescript-eslint/no-non-null-asserted-nullish-coalescing': 'error',
+      'no-throw-literal': 'off',
```

Comment:
k, disable the non-ts one, and enable the ts-specific one, this makes sense to leave here.